### PR TITLE
impr: update unregistered solicitor email (FPLA-2984)

### PR DESCRIPTION
**Pull request by lewisbirks**
### JIRA link (if applicable) ###

[FPLA-2984](https://tools.hmcts.net/jira/browse/FPLA-2984)

### Change description ###

Update the unregistered solicitor email and introduce corresponding template.

Notification: [68b071c5-2eb9-4646-9d3e-22cc168477cf](https://www.notifications.service.gov.uk/services/12f756df-f01d-4a32-a405-e1ea8a494fbb/templates/68b071c5-2eb9-4646-9d3e-22cc168477cf)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/respondentsolicitor/UnregisteredRespondentSolicitorTemplate.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.fpl.model.notify.respondentsolicitor;
+
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.hmcts.reform.fpl.model.notify.NotifyData;
+
+@Value
+@Builder
+public class UnregisteredRespondentSolicitorTemplate implements NotifyData {
+    String ccdCode;
```
Comment (joez-f) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/respondentsolicitor/UnregisteredRespondentSolicitorTemplate.java:10]: very minor: maybe ccdCaseNumber or something else (dont like ccdCode, but not sure why)

Comment (lewisbirks) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/respondentsolicitor/UnregisteredRespondentSolicitorTemplate.java:10]: this is what was in the template, I can change the template if you want it to match `ccdCaseNumber` or just add a json property annotation and change the field name.

Comment (lewisbirks) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/respondentsolicitor/UnregisteredRespondentSolicitorTemplate.java:10]: Changed to ccdNumber, consistent with the NoC emails

**Review by joez-f — APPROVED** [review]
lgtm

**Pull request merged**